microprofile-open-api/src/test/java/org/jboss/eap/qe/microprofile/openapi/v20/MicroProfileOpenApi20Test.java

```diff
@@ -0,0 +1,224 @@
+                schemaProperties.containsKey("alias"));
+
+        Assert.assertTrue("", ((LinkedHashMap) schemaProperties.get("alias")).containsValue("Alias of district"));
+
+        // 2.0 The @RequestBodySchema annotation has been added to provide a shorthand mechanism to specify the schema for a request body
+        Map<String, Object> getDistrictByCode20Path = (Map<String, Object>) paths.get("/districts/{code}/v20");
+        Assert.assertFalse("\"/districts/{code20}\" property is empty", getDistrictByCode20Path.isEmpty());
+
+        Map<String, Object> patchMethod = (Map<String, Object>) getDistrictByCodePath.get("patch");
+        Assert.assertFalse("\"/districts/{code20}\" \"patch\" property is empty", patchMethod.isEmpty());
+        Assert.assertEquals("\"/districts/{code20}\" \"patch\" property should have exactly 13 keys",
+                13, patchMethod.keySet().size());
+
+        Map<String, Object> requestBody = (Map<String, Object>) patchMethod.get("requestBody");
+        Assert.assertFalse("\"/districts/{code20}\" \"requestBody\" for PATCH verb is empty", requestBody.isEmpty());
+
+        Map<String, Object> content = (Map<String, Object>) requestBody.get("content");
+        Assert.assertFalse(
+                "\"/districts/{code20}\" \"requestBody\" for PATCH verb has empty \"content\" property",
+                content.isEmpty());
+
+        Map<String, Object> contentJson = (Map<String, Object>) content.get("application/json");
+        Assert.assertFalse(
+                "\"/districts/{code20}\" \"response\" for PATCH verb has \"content\" but empty \"application/json\" property",
+                contentJson.isEmpty());
+
+        Map<String, Object> requestBodySchema = (Map<String, Object>) contentJson.get("schema");
+        Assert.assertFalse(
+                "\"/districts/{code20}\" \"response\" for GET verb has \"application/json\" but empty \"schema\" property",
+                requestBodySchema.isEmpty());
+
+        Assert.assertEquals("Specified schema for the request body is not equal to DistrictEntity",
+                "#/components/schemas/DistrictEntity", requestBodySchema.get("$ref"));
+
+        // 2.0 The @APIResponseSchema annotation has been added to provide a shorthand mechanism to specify the schema for a response body
+        Map<String, Object> patchResponses = (Map<String, Object>) patchMethod.get("responses");
+        Assert.assertFalse("\"/districts/{code20}\" \"responses\" for PATCH verb is empty", patchResponses.isEmpty());
+
+        Map<String, Object> patchHttp200Response = (Map<String, Object>) patchResponses.get("200");
+        Assert.assertFalse("\"/districts/{code20}\" \"response\" for PATCH verb and HTTP status 200 is empty",
+                patchHttp200Response.isEmpty());
+
+        Map<String, Object> patchContentAnnotation = (Map<String, Object>) patchHttp200Response.get("content");
```

Comment:
I'd name this `pathResponseContentAnnotation`, WDYT? See similar thoughts in previous comments regarding variables' names.